docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
       - ./frontend:/app:delegated
       - ./common:/common:delegated
       - ./.cache/npm:/home/node/.npm/_cacache:delegated
+      - ./.cache/npm/sentry-cli:/home/node/.npm/sentry-cli:delegated
```

Comment:
Isn't this already included in the line above?

Reply:
no, there the npm directory only contains contents of the _cacache directory and the .npm is mounted into the .npm/_cacache directory.
Now sentry-cli is also mounted in _cacache, but i think we can live with that